source/Calamari.Common/CalamariFlavourProgramAsync.cs

```diff
@@ -130,10 +132,11 @@ protected async Task<int> Run(string[] args)
 
         IEnumerable<Assembly> GetAllAssembliesToRegister()
         {
-            var programAssembly = GetProgramAssemblyToRegister();
+            var programAssembly = GetProgramAssembliesToRegister();
```

Comment:
This way Calamari.AzureServiceFabric will be able to just add Calamari.Scripting and have it registered automatically.

The alternative is that everyone registers manually and we create methods to make it easier to register a command etc.

Reply:
why don't we just scan all assemblies loaded into the AppDomain?

Reply:
We can also do that, I'm not fussed. 

All I care about is that `Calamari.AzureServiceFabric` is treated the same way as `Calamari.Scripting`.